### PR TITLE
DU for the verb classes (#590)

### PR DESCRIPTION
**Pull request by psfinaki**
I made DU for the verb classes as discussed in one of the recent PRs.
Finally I decided to give them some names which is better than numbers. Names refer to the verb ending in the third person singular, as per the [official grammar](https://cs.wikipedia.org/wiki/%C4%8Cesk%C3%A1_slovesa#Slovesn%C3%A9_t%C5%99%C3%ADdy_a_vzory).

### CODE REVIEW

File: src/Client/pages/VerbImperatives.fs [hunk 1]
```diff
@@ -20,16 +20,16 @@ type Msg =
     | Task of Task.Msg
     
 let patterns =
-    dict [ (1, ["nést"; "číst"; "péct"; "třít"; "brát"; "mazat"])
-           (2, ["tisknout"; "minout"; "začít"])
-           (3, ["krýt"; "kupovat"])
-           (4, ["prosit"; "čistit"; "trpět"; "sázet"])
-           (5, ["dělat"]) ]
+    dict [ (E, ["nést"; "číst"; "péct"; "třít"; "brát"; "mazat"])
+           (NE, ["tisknout"; "minout"; "začít"])
```
Comment (reflash) [inline src/Client/pages/VerbImperatives.fs]: One question: how do we distinguish between `začít` and `třít` ? (same question regarding irregular verbs, which have shape of one class but conjugate as a different one)

File: src/Client/pages/VerbImperatives.fs [hunk 2]
```diff
@@ -20,16 +20,16 @@ type Msg =
...
+           (JE, ["krýt"; "kupovat"])
+           (Í, ["prosit"; "čistit"; "trpět"; "sázet"])
+           (Á, ["dělat"]) ]
 
 let getPatterns ``class`` = patterns.[``class``]
 
 let getTask ``class`` pattern =
-    let classQuery = ``class`` |> Option.map (sprintf "class=%i")
+    let classQuery = ``class`` |> Option.map string |> Option.map (sprintf "class=%s")
```
Comment (reflash) [inline src/Client/pages/VerbImperatives.fs]: is it to display a tuple? I think %O or %A should work

Comment (psfinaki) [reply src/Client/pages/VerbImperatives.fs]: It is for DU (the `VerbClass`) but yes, `%A` is what's needed - thanks. Made it for Gender at other pages as well.

File: src/Client/pages/VerbImperatives.fs (hunk 1 above)
Comment (psfinaki) [reply src/Client/pages/VerbImperatives.fs]: Well, short and sad answer - we don't. We detect the class not by the infinitive but by the third person singular (_začne_, _tře_)- the names of the classes refer to those endings. 

So just by infinitive one cannot for sure know the imperative of the word - they need to know the third person singular first, so they need to know some of the conjugation anyway. But this is ~~life~~Czech.

I should finally write rules in this exercise and explain things a bit.

**Pull request merged**